Changelog.md

```diff
@@ -2,7 +2,69 @@
 
 ### Gazebo Rendering 10.X
 
-### Gazebo Rendering 10.0.0 (20XX-XX-XX)
+### Gazebo Rendering 10.0.0 (2025-09-XX)
+
+1. **Baseline:** this includes all changes from 9.3.0 and earlier.
+
+1. CppCheck fixes
+    * [Pull request #1148](https://github.com/gazebosim/gz-rendering/pull/1148)
+
+1. Enable doxygen CI check on noble
+    * [Pull request #1129](https://github.com/gazebosim/gz-rendering/pull/1129)
+
+1. Remove symlinks of versioned to unversioned engine libraries
+    * [Pull request #1142](https://github.com/gazebosim/gz-rendering/pull/1142)
+
+1. Fix user settings of thermal camera being ignored
+    * [Pull request #1138](https://github.com/gazebosim/gz-rendering/pull/1138)
+
+1. Disable a couple of tests on windows
+    * [Pull request #1126](https://github.com/gazebosim/gz-rendering/pull/1126)
+
+1. Fix clamp of temperature in thermal camera output
+    * [Pull request #1136](https://github.com/gazebosim/gz-rendering/pull/1136)
+
+1. Bump gz-cmake and others in jetty
+    * [Pull request #1131](https://github.com/gazebosim/gz-rendering/pull/1131)
+
+1. Add sanity check for valid submesh indices
+    * [Pull request #1123](https://github.com/gazebosim/gz-rendering/pull/1123)
+
+1. Remove unused variable in OgreRenderingEngine.cc
+    * [Pull request #1119](https://github.com/gazebosim/gz-rendering/pull/1119)
+
+1. Remove ClearMaterialsCache call when a material is destroyed in ogre2
+    * [Pull request #1110](https://github.com/gazebosim/gz-rendering/pull/1110)
```

Comment:
In 9.2.0

Reply:
removed. [3712675](https://github.com/gazebosim/gz-rendering/pull/1158/commits/37126756b96db6f57df30ffe7fd5c2189a644a44)